### PR TITLE
info box

### PR DESCRIPTION
**Pull request by kachakot**

### CODE REVIEW

File: src/components/MainNav/MainNav.stories.js [hunk 1]
```diff
@@ -4,8 +4,7 @@ import { storiesOf } from '@storybook/react'
 import MainNav from './MainNav'
 
 storiesOf('MainNav', module)
+  .addDecorator(storyFn => <div style={{ background: '#000', padding: '5px' }}>{storyFn()}</div>)
```
Comment (rgolubowicz) [inline src/components/MainNav/MainNav.stories.js]: break it in a few lines to make it more readable

File: src/components/Field/Field.stories.js [hunk 2]
```diff
@@ -12,23 +12,20 @@ storiesOf('Field', module)
       type="text"
     />
   ))
+  .addDecorator(storyFn => <div style={{ background: '#000', padding: '5px' }}>{storyFn()}</div>)
```
Comment (rgolubowicz) [inline src/components/Field/Field.stories.js]: break it into a few lines to make it more readable

File: src/components/MainNav/MainNav.stories.js (hunk 1 above)
Comment (kachakot) [reply src/components/MainNav/MainNav.stories.js]: done

File: src/components/Field/Field.stories.js (hunk 2 above)
Comment (kachakot) [reply src/components/Field/Field.stories.js]: done

**Pull request merged**